build/int.cloudbuild.yaml

```diff
@@ -81,6 +81,29 @@ steps:
     args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do destroy mysql-public-local']
     timeout: 1800s
 
+  - id: go-init-mysql-public
+    waitFor:
+      - destroy-mysql-public-local
```

Comment:
@morgante suggested we remove the older ruby tests as we swap these out with newer tests

Reply:
Acked